scenarios/basics/fleetwise/SPECIFICATION.md

```diff
@@ -0,0 +1,314 @@
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+8. Create a vehicle
+Creating a vehicle in AWS IoT FleetWise allows you to digitally
+represent and manage a physical vehicle within the AWS ecosystem.
+This enables efficient ingestion, transformation, and transmission
+of vehicle telemetry data to the cloud for analysis.
+
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Vehicle 'vehicle60' created successfully.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+9. Display vehicle details
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Vehicle Details:
+• modelManifestArn : arn:aws:iotfleetwise:us-east-1:814548047983:model-manifest/manifest60
+• vehicleName : vehicle60
+• creationTime : 2025-04-29T16:00:02.147Z
+• lastModificationTime : 2025-04-29T16:00:02.147Z
+• decoderManifestArn : arn:aws:iotfleetwise:us-east-1:814548047983:decoder-manifest/decManifest60
+• attributes : {}
+• arn : arn:aws:iotfleetwise:us-east-1:814548047983:vehicle/vehicle60
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+10. Delete the AWS IoT Fleetwise Assets
+Would you like to delete the IoT Fleetwise Assets? (y/n)
+y
+vehicle60 was successfully deleted
+decManifest60 was successfully deleted
+manifest60 was successfully deleted
+fleet60 was successfully deleted
+catalog60 was successfully deleted
+--------------------------------------------------------------------------------
+Thank you for checking out the AWS IoT Fleetwise Service Use demo. We hope you
+learned something new, or got some inspiration for your own apps today.
+For more AWS code examples, have a look at:
+https://docs.aws.amazon.com/code-library/latest/ug/what-is-code-library.html
+
+--------------------------------------------------------------------------------
+```
+
+## SOS Tags
+
+The following table describes the metadata used in this Basics Scenario.
+
+
+| action                 | metadata file              | metadata key                         |
+|------------------------|----------------------------|------------------------------------- |
+|`createSignalCatalog`   |iot_fleetwise_metadata.yaml |iotfleetwise_CreateSignalCatalog      |
+|`createFleet`           |iot_fleetwise_metadata.yaml |iotfleetwise_CreateFleet              |
+|`createGateway `        |iot_fleetwise_metadata.yaml |iotfleetwise_CreateGateway            |
```

Comment:
What is CreateGateway? I don't see it in any of the code or metadata.

Reply:
Fixed by removing it